experiments/pages/docusign/autoPopulationTutorial.md

```diff
@@ -0,0 +1,135 @@
+Because DocuSign’s REST API is fairly complex, creating a custom request body with merge variables is the way to go. (View an example below!)
+You can basically follow the format given in the DocuSign API Explorer:  https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=Envelopes&operations=create but you will only need the following fields: 
+1.	Envelope
+a.	Status  - either sent or created (for a draft)
+b.	Template Id  - referenced above. 
+c.	Email Subject - define the subject of the email to be sent with the docusign doc. 
+2.	Recipient
+a.	name
+b.	Email
+c.	roleName - same as defined in the DocuSign template
+d.	recipientId - the best way to get this is to do a get request on the api explorer  (https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=EnvelopeRecipients&operations=list) 
+e.	routingOrder (optional) - can also be defined in DocuSign, but it can be helpful to conditionally choose the routing order in Skuid. 
+3.	Recipient tab
+a.	tabLabel - Data Label defined in DocuSign
+b.	value - the value you want to insert (use merge syntax here!)
+c.	font (optional) - choose a font to stylize your text. 
+i.	Supported Fonts: Arial, Arial, ArialNarrow, Calibri, CourierNew, Garamond, Georgia, Helvetica,   LucidaConsole, Tahoma, TimesNewRoman, Trebuchet, Verdana, MSGothic, MSMincho, Default.
+
+##### See it in action!: 
+1.	In your Skuid page:
+2.	Make sure there is a new row on the “POST” model. 
+3.	Create an action that saves the new row. 
+4.	Look for an email from DocuSign (if you are the first recipient) 
+5.	You did it!
+
+##### Promote your key to a production account: 
+In order to send real DocuSign documents, you must promote your API key to a production account. Once it has been promoted, you will need to change some of the data source settings to match the new URL etc. 
+To do this:
+1.	Promote your key (tutorial:https://support.docusign.com/guides/ndse-admin-guide-api-and-keys) 
+2.	With the new key in the production account, follow steps 2-5 under the DocuSign section above. 
+3.	You will also need to transfer your template, which is made simple by selecting the download option on the template you want to transfer and then selecting new > upload template in the production account. 
+4.	In Skuid configure, you will need to modify the authentication provider and data source
+a.	New Data source URL: https://na2.docusign.net/restapi/v2 (may change based on your instance)
+b.	New Authentication Provider endpoint and token endpoint URLs : https://account.docusign.com/oauth/auth
+c.	Also make sure to update the client Id and Client secret if they have changed. 
+5.	In your Skuid page, modify the JS snippet to change the url in the request body to: https://na2.docusign.net/restapi/v2/accounts/<accountId>/envelopes/ 
+6.	Test it out! 
+
+##### Troubleshooting:
+1.	To see the status of your request in your Skuid page, go to inspect > network and look for the request. 
```

Comment:
Maybe to clarify you could just suggest using the browser's network inspector, this will be a little different depending on which browser you're using

Reply:
Good suggestion.